scm/driver/stash/util.go

```diff
@@ -108,10 +108,12 @@ func encodePullRequestListOptions(opts scm.PullRequestListOptions) string {
 	if opts.Size != 0 {
 		params.Set("limit", strconv.Itoa(opts.Size))
 	}
-	if opts.Open && opts.Closed {
+	if opts.Open && opts.Closed && opts.Merged {
```

Comment:
of what i understand we need state all -> lets not make too many changes and in caller add both open and close

Reply:
did this according to the changes already present with this condition
`if opts.Open && opts.Closed { params.Set("state", "all") }`

Reply:
fixed